CONTRIBUTING.md

```diff
@@ -6,3 +6,11 @@ If you want to contribute please follow this guideline.
 
```

Comment:
unrelated to this file:

- can we use a Seach icon in the Search field?
- Should we use a + icon on the "add new project" button?

Reply:
can put search Icon infront of search field

could use + icon , no big deifference